Pilot_Plant_Dataset_Doc.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<p>Please don't strangle me. But. Could we use a title that reflects either Del Mar or Distillation Column or Glycol (I think)? When I talk about the dataset, I usually use some or all of those words, but I don't think Pilot Plant is a strong differentiator or descriptive of what the dataset contains at first glance.</p><p> </p><p>Annnnd now that I have read further down, I see you use Del Mar College Glycol Pilot Plant. Can we use that in the title as well? Or at least Glycol Pilot Plant?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/academic-hub/datasets/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done